tests/application/cms/test_models.py

```diff
@@ -717,11 +802,11 @@ def test_get_pages_by_type_returns_pages_ordered_by_title_and_version(
         ),
     )
     def test_area_covered_formatter(self, countries, formatted_string):
-        page = MeasureVersion(guid="test_page", version="1.0", area_covered=countries)
+        measure_version = MeasureVersionFactory(guid="test_page", version="1.0", area_covered=countries)
```

Comment:
I don't think we want/need to specify guid any more?

Reply:
Hmm, seems like ordeing measures and the review tokens still depend on guid.

I'll make a note of these on the "clean-up" ticket to adjust.